docs/build/build-guide.md

```diff
@@ -7,17 +7,51 @@ keywords: [build, how to build, develop, parachain, smart contract]
 slug: ../build-guide
 ---
 
+import Tabs from "@theme/Tabs";
+
+import TabItem from "@theme/TabItem";
+
 Polkadot is a blockchain protocol with two goals: providing **shared security** among all connected
 parachains and allowing all connected chains to **interoperate** by using
 [XCM](../learn/learn-xcm.md). With the advent of
-[PDKs](../build/build-parachains.md#parachain-development-kit) like Parity Substrate and Cumulus,
-the time it takes to develop and launch a new chain has dropped significantly. While before it would
-take years to launch a new chain, now it may only take weeks or even days.
+[PDKs](../build/build-parachains.md#parachain-development-kit) like
+[Substrate](https://substrate.io/) and [Cumulus](https://github.com/paritytech/cumulus), the time it
+takes to develop and launch a new chain has dropped significantly. While before it would take years
+to launch a new chain, now it may only take weeks or even days depending on your goals.
 
 This guide will walk you through the steps you can take today to get started building your vision
 with {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}. It will explain the difference
-between a parachain and a smart contract (and why one may be better suited for your application over
-the other).
+between a [parachain](../learn/learn-parachains.md) and a smart contract in terms of deciding which
+best fits your architectural needs.
+
+This build guide provides four different tracks:
+
+1. [Building Parachains](#building-parachains)
+2. [Building a Pallet](#building-a-pallet)
+3. [Developing Smart Contracts](#developing-smart-contracts)
+4. [Developing a dApp](#developing-a-dapp)
+
+:::info
+
+Keep in mind that these tracks are merely suggestive, and there are many ways to utilize Polkadot,
+Substrate, and their various developmental components. For more inspiration, take a look at the
+[open source projects](./build-open-source.md) featured here in the wiki!
+
+:::
+
+Before diving into any one of these tracks, it is encouraged to read about Polkadot and its networks
+in order to gain context about the application you could make.
+
+## Development Ecosystem Overview
+
+![Path to Building a Parachain](../assets/parachain-development.png)
```

Comment:
I find this figure a bit convoluted. Especially the lines linking things to the cumulus box is unclear to me. Can you explain further?

Reply:
Actually, it seems a lot of that repo may be relevant here!  I'll probably tag you in some of the diagrams I'm putting together over there :) Mostly to do with `frame` and their relationship with Substrate core libraries, that kind of stuff

Reply:
Working diagram - I think it's clearer than the other one so far

![Screenshot 2023-06-22 at 10 20 07 AM](https://github.com/w3f/polkadot-wiki/assets/26698074/f2b69ce1-b265-4a39-9470-4e5163e72aee)

Reply:
If something like this is good, then I can theme it

Reply:
I am seeing this with a lot of delay, but the outcome is great, thanks!